ignite/handlers/checkpoint.py

```diff
@@ -229,13 +230,11 @@ class Checkpoint(Serializable):
             # Run evaluation on epoch completed event
             # ...
 
-            score_function = Checkpoint.get_default_score_fn("accuracy")
-
             to_save = {'model': model}
             handler = Checkpoint(
                 to_save, DiskSaver('/tmp/models', create_dir=True),
                 n_saved=2, filename_prefix='best',
-                score_function=score_function, score_name="val_acc",
+                score_name="accuracy",
```

Comment:
@KickItLikeShika please update this line as well: https://github.com/pytorch/ignite/pull/2146/files#diff-26b2fc842f0567d57c38661f9ef773fe32de32caa2b5e25e9043fcb91238774dR244
```
> ["best_model_9_val_acc=0.77.pt", "best_model_10_val_acc=0.78.pt", ]
```

We have to explain above that evaluator.state.metrics will contain "accuracy" metric which will be used here